### PR TITLE
[DLRM_DCNV2] Training v3.0 RCPs

### PR DESCRIPTION
**Pull request by ShriyaPalsamudram**
Update to dlrm_dcnv2 RCPs for training 3.0 submission

### CODE REVIEW

**Comment by xyhuang** [comment]
@erichan1 is training WG good with merging this? thx

**Comment by erichan1** [comment]
Generally lgtm.

@ShriyaPalsamudram Just so I know, who ran these? Just so we know who to talk to if questions come up. cc @janekl

**Review by erichan1 — APPROVED** [review]

**Comment by janekl** [comment]
> Generally lgtm.
> 
> @ShriyaPalsamudram Just so I know, who ran these? Just so we know who to talk to if questions come up. cc @janekl

Hi @erichan1. We ran this using the same command as in https://github.com/mlcommons/logging/pull/300 and https://github.com/mlcommons/logging/pull/305 just changing batch size and learning rate accordingly. Feel free to contact me or Shriya in case of any questions.
